src/index.ts

```diff
@@ -25,6 +25,24 @@ export interface Collection<K, V> extends Map<K, V> {
 export class Collection<K, V> extends Map<K, V> {
 	public static readonly default: typeof Collection = Collection;
 
+	/**
+	 * Obtains an element if the key exists, otherwise sets and returns the value provided by the default value generator.
+	 *
+	 * @param {*} key - Key to ensure (if unset) or return from the collection (if set)
```

Comment:
```suggestion
	 * @param {*} key - The key to get from if it exists or set otherwise.
```

I'm unsure about this wording, cc: @vladfrangu, @almostSouji

Reply:
What's wrong with the previous wording?

Reply:
It has a lot of redundancy ("if set" and "if unset") as sounds a bit heavy to read.